vscode/src/codeLens.ts

```diff
@@ -51,14 +60,25 @@ function mapCodeLens(cl: ICodeLens): vscode.CodeLens {
       command = "qsharp-vscode.runEditorContents";
       tooltip = "Run program";
       break;
-    default:
-      throw new Error(`Unknown code lens command: ${cl.command}`);
+    case "circuit":
+      if (
+        !vscode.workspace.getConfiguration("Q#").get<boolean>("featureCircuits")
```

Comment:
Is there any UX for this settings (by declaring it in the package.json)? Or do we want/expect users to manually put this in the JSON file to enable it?

Reply:
Yes, I thought we may want to "hide" the feature for now, though on second thought I don't know why I thought that :) . I'll add a proper description for it in package.json.